models/configuration.go

```diff
@@ -1,35 +1,12 @@
 package models
 
-import (
-	"crypto/x509"
-	"errors"
-	"fmt"
-)
-
-// NewConfiguration is the way a Configuration is intended to be obtained. It ensures the
-// given certificates are valid and prepares a CA certificate pool to be used for client
-// certificate verification.
-func NewConfiguration(certificates []string, pcfAPIAddr, pcfUsername, pcfPassword string) (*Configuration, error) {
```

Comment:
I ended up not needing this constructor because I found I could eliminate setting up the `verifyOpts` in advance. I liked stripping it because that's more go-y anyways.